src/openms_gui/source/VISUAL/TOPPViewSpectraViewBehavior.cpp

```diff
@@ -324,6 +324,16 @@ namespace OpenMS
           peak1d.setIntensity(cpeak.getIntensity());
           spectrum.push_back(peak1d);
         }
+        
+        // Add at least one data point to the chromatogram, otherwise "addLayer" will fail and a segfault occurs later
```

Comment:
yes, but I am not experienced enough with the code to do that at the moment. It seems that 
` bool Spectrum1DCanvas::finishAdding_()` doesnt really deal with it at all but just outputs an error and returns false and then `TOPPViewSpectraViewBehavior::activate1DSpectrum` returns. This, however, does not seem to leave things in a good state.

https://github.com/OpenMS/OpenMS/blob/develop/src/openms_gui/source/VISUAL/Spectrum1DCanvas.cpp#L1025